main/main.cpp

```diff
@@ -1956,6 +1956,9 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 		GLOBAL_DEF_RST_NOVAL(PropertyInfo(Variant::STRING, "rendering/rendering_device/driver.android", PROPERTY_HINT_ENUM, driver_hints), default_driver);
 		GLOBAL_DEF_RST_NOVAL(PropertyInfo(Variant::STRING, "rendering/rendering_device/driver.ios", PROPERTY_HINT_ENUM, driver_hints), default_driver);
 		GLOBAL_DEF_RST_NOVAL(PropertyInfo(Variant::STRING, "rendering/rendering_device/driver.macos", PROPERTY_HINT_ENUM, driver_hints), default_driver);
+
+		GLOBAL_DEF_RST("rendering/rendering_device/vulkan/fallback_to_vulkan", true);
+		GLOBAL_DEF_RST("rendering/rendering_device/d3d12/fallback_to_d3d12", true);
```

Comment:
Wouldn't it be clearer to have it named `rendering/rendering_device/d3d12/fallback_to_vulkan` and `rendering/rendering_device/vulkan/fallback_to_d3d12`, since it's Direct3D falling back to Vulkan and vice versa? Or have it directly under the `rendering/rendering_device/`

Reply:
Moved to `rendering/rendering_device/`